be/src/formats/orc/orc_mapping.cpp

```diff
@@ -117,6 +117,26 @@ StatusOr<std::unique_ptr<OrcMapping>> OrcMappingFactory::build_mapping(
     return orc_mapping;
 }
 
+Status OrcMappingFactory::_check_orc_type_can_converte_2_logical_type(const orc::Type& orc_source_type,
+                                                                      const TypeDescriptor& slot_target_type) {
+    bool can_convert = true;
```

Comment:
This should be `false`, right?

Reply:
We only check for complex types now, if `orc_source_type` is a primitive type, we will not check it. So we need set `can_convert=true` by default.